wallet/core/src/storage/local/wallet.rs

```diff
@@ -66,7 +57,8 @@ impl Wallet {
 
     pub async fn try_store(&self, store: &Storage) -> Result<()> {
         store.ensure_dir().await?;
-        fs::write_json(store.filename(), self).await?;
+        let serialized = BorshSerialize::try_to_vec(self)?;
+        fs::write(store.filename(), serialized.as_slice()).await?;
```

Comment:
```
let mut file = File::create(store.filename(), )?;
BorshSerialize::serialize(self,  &mut file)
```

it can avoid creating of temp buffer

Reply:
That's not how it works :). Wrong fs :). (And no, not changing the module name :)).  Although maybe I should fully qualify it or "as" it.

OTOH I could cfg it and make it use native fs on native. Need to think about it.

Reply:
Refactored using platform-specific behavior: https://github.com/aspectron/rusty-kaspa/commit/bf4ffbe0ea03087a04cb206ef09a6b4e62c46985